tests/mapped_struct.py

```diff
@@ -2113,9 +2113,24 @@ def testHashFloats(self):
     def testHashStrings(self):
         self.assertHashesOK(("abcdef", "123456789", "!@#%&$"))
 
+    def testHashStringsFixed(self):
+        values_hashes = (
+            ("abcdef", 18053520794346263629L),
+            ("123456789", 10139926970967174787L),
+            ("!@#%&$", 15648343848775299486L))
```

Comment:
PEP formatting:

``` python
(
    (blah),
    (blah),
    (blah),
)
```

Ie: trailing comma on the last entry, closing paren on its own line